google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractStructReader.java

```diff
@@ -69,6 +71,8 @@ protected Value getValueInternal(int columnIndex) {
 
   protected abstract List<String> getStringListInternal(int columnIndex);
 
+  protected abstract List<String> getJsonListInternal(int columnIndex);
```

Comment:
```suggestion
  protected String getJsonListInternal(int columnIndex) {
    throw new UnsupportedOperationException("Not implemented");
  }
```

Reply:
We should provide a default implementation not to break any users who have their own implementation of the interface.